### ISSUE TITLE
Machine Learning

### ISSUE THREAD

**Issue by chochinlu** [chochinlu/blog#72]
## Learning Rates 

[reference](https://towardsdatascience.com/understanding-learning-rates-and-how-it-improves-performance-in-deep-learning-d0d4059c1c10)
- What is the learning rate? What is it’s significance?
- How does one systematically arrive at a good learning rate?
- Why do we change the learning rate during training?
- How do we deal with learning rates when using pretrained model?
